config/routes.rb

```diff
@@ -8,7 +8,7 @@
 
   post '/locks/from_pool(.:format)' => 'locks#lock_from_pool', as: "create_from_pool"
 
+  match '/pools/disable(.:format)' => 'pools#disable', via: [:put, :patch], as: "disable_pool"
   resources :pools
```

Comment:
@akostadinov I was confused here a bit, when I'm changing the order of [these two lines](https://github.com/maybelinot/ownthat/blob/4fdbbd156a0ff0a75e48a9c96dfd6c52c33f4415/config/routes.rb#L11-L12) request on `/pools/disable` redirected to pools#update method..

Will need more time to become familiar with all RoR stuff..